pkg/k8s/secrets.go

```diff
@@ -187,6 +178,20 @@ func (c secretClient) validateSecret(secret types.Secret) error {
 	return nil
 }
 
+func (c secretClient) getValidSecretData(secret types.Secret) map[string][]byte {
+	var data map[string][]byte
+	if len(secret.RawValue) > 0 {
+		data = map[string][]byte{
+			secret.Name: secret.RawValue,
+		}
+	} else {
+		data = map[string][]byte{
+			secret.Name: []byte(secret.Value),
+		}
+	}
+	return data
+}
```

Comment:
Early returns, like below, are generally preferred to if/else, you can even remove the placeholder `data` variable in this case 
```suggestion
	if len(secret.RawValue) > 0 {
		return map[string][]byte{
			secret.Name: secret.RawValue,
		}
	}
	
	return map[string][]byte{
		secret.Name: []byte(secret.Value),
	}
}
```

Reply:
Thanks for the feedback.